src/browser/components/TabNavigation/Navigation.tsx

```diff
@@ -156,48 +159,47 @@ class Navigation extends Component<NavigationProps, NavigationState> {
         )
       })
 
-    const getContentToShow = (openDrawer?: null | string) => {
-      if (openDrawer) {
+    const getContentToShow = (selectedDrawerName?: null | string) => {
+      if (selectedDrawerName) {
         const filteredList = topNavItems
           .concat(bottomNavItems)
-          .filter(item => item.name.toLowerCase() === openDrawer)
+          .filter(item => item.name.toLowerCase() === selectedDrawerName)
         const TabContent = filteredList[0].content
         return <TabContent />
       }
       return null
     }
-    const topNavItemsList = buildNavList(topNavItems, this.state.drawerContent)
+    const topNavItemsList = buildNavList(
+      topNavItems,
+      this.props.selectedDrawerName
+    )
     const bottomNavItemsList = buildNavList(
       bottomNavItems,
-      this.state.drawerContent
+      this.props.selectedDrawerName
     )
 
+    const isDrawerVisible = () => this.state.transitionState !== Closed
```

Comment:
We could simplify this to be a constant rather than a function, right? `const drawerIsVisible = this.state.transitionState !== Closed`. WDYT?